docker-storage-setup.sh

```diff
@@ -379,27 +360,25 @@ create_data_lv() {
 
   check_min_data_size_condition
 
-  # TODO: Error handling when DATA_SIZE > available space.
-  if [[ $DATA_SIZE == *%* ]]; then
-    lvcreate -y -l $DATA_SIZE -n $DATA_LV_NAME $VG
-  else
-    lvcreate -y -L $DATA_SIZE -n $DATA_LV_NAME $VG
-  fi
-}
-
-create_lvm_thin_pool () {
-  if [ -z "$DEVS_ABS" ] && [ -z "$VG_EXISTS" ]; then
-    Fatal "Specified volume group $VG does not exist, and no devices were specified"
+  # Calculate size of metadata lv. Reserve 0.1% of the free space in the VG
+  # for docker metadata.
+  VG_SIZE=$(vgs --noheadings --nosuffix --units s -o vg_size $VG)
+  META_SIZE=$(( $VG_SIZE / 1000 + 1 ))
+  if [ -z "$META_SIZE" ];then
+    Fatal "Failed to calculate size of metadata volume"
```

Comment:
we don't need to do any of the above. let LVM do this for us as it will automatically create the metadata LV for us. 

If we want to allow the user to specify META_SIZE in the future, then we can by simply exposing the `--poolmetadatasize` argument to them.